server/cmwell-it/src/it/scala/cmwell/it/AggregationTests.scala

```diff
@@ -0,0 +1,104 @@
+/**
+  * Copyright 2015 Thomson Reuters
+  *
+  * Licensed under the Apache License, Version 2.0 (the “License”); you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+  * an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  *
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+
+package cmwell.it
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.{AsyncFunSpec, Inspectors, Matchers}
+import play.api.libs.json.{JsValue, _}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.util.{Failure, Success, Try}
+
+class AggregationTests extends AsyncFunSpec with Matchers with Inspectors with Helpers with fixture.NSHashesAndPrefixes with LazyLogging {
+
+  describe("Agg API should") {
+    //Assertions
+    val ingestGeonames = Future.traverse(Seq(293846)) { n =>
+      val agg = scala.io.Source.fromURL(this.getClass.getResource(s"/agg/aggnames_$n.nq")).mkString
```

Comment:
resource leak! You should close the the resource. Please dismiss this remark if it is fixed in scala 2.12.